COVID19-Event-Risk-Planner/makeDailyMaps.R

```diff
@@ -23,27 +25,132 @@ print(current_time)
 
 getData <- function() {
     dataurl <- "https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv"
-  data <- read.csv(dataurl, stringsAsFactors = FALSE) %>% mutate(date = as_date(date))
+  data <- vroom(dataurl)
   county <<- st_read("map_data/geomUnitedStates.geojson")
+  countyVacc <<- st_read("map_data/geomUnitedStatesCV.geojson")
   stateline <<- st_read("map_data/US_stateLines.geojson")[,c('STUSPS','NAME')]
   names(stateline) <- c('stname','name')
-  pop <- read.csv("map_data/county-population.csv", stringsAsFactors = FALSE)
+  pop <- vroom("map_data/county_population.csv")
+  popC <- vroom("map_data/county_populationCV.csv")
+  pop <- pop%>%
+      full_join(
+          select(popC, c(fips, popC=pop)), by="fips"
+      )
+  
+
+  
   cur_date <- ymd(gsub("-", "", Sys.Date())) - 1
   past_date <- ymd(cur_date) - 14
-  vacc_data <- read.csv("https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/vaccinations/us_state_vaccinations.csv",stringsAsFactors = FALSE)
-  VaccImm <<- vacc_data[which(vacc_data$date==past_date),] %>% select(location,pct_partially_vacc = people_vaccinated_per_hundred,pct_fully_vacc = people_fully_vaccinated_per_hundred)
-  VaccImm$location[which(VaccImm$location=="New York State")] <<- "New York"
+  # vaccine
+  nyc <- c(36005, 36047, 36061, 36085, 36081)
+  vacc_data <- vroom("https://raw.githubusercontent.com/bansallab/vaccinetracking/main/vacc_data/data_county_timeseries.csv")
+  vacc_data <- vacc_data%>%
+      # filter(CASE_TYPE %in% c("Complete","Partial"))%>%
+      filter(CASE_TYPE %in% c("Complete"))%>%
+      pivot_wider(
+          names_from = CASE_TYPE,
+          values_from = CASES
+      )%>%
+      select(
+          county = COUNTY,
+          cnt_fully_vacc = Complete,
+          # cnt_partially_vacc = Partial, # removing partials
+          date = DATE
+      )%>%
+      mutate(
+          
+          date = as_date(date),
+          county = case_when(
+              as.numeric(county) %in% bb_lp ~ 2997,
```

Comment:
@freyja-bt `bb_lp` and `y_ha` refer to objects that don't exist

Reply:
I'll replace them

Reply:
I made a new PR with the change